src/ducks/_Table/index.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 import cx from 'classnames'
 import { FluidSkeleton as Skeleton } from '../../components/Skeleton'
-import NoDataImage from '../../components/Illustrations/NoData'
+import EmptySection from '../../pages/Watchlists/EmptySection'
+import { BLOCKCHAIN_ADDRESS } from '../../ducks/Watchlists/detector'
+import { DesktopOnly } from '../../components/Responsive'
```

Comment:
This is general component and should not include any specific implementation dependencies.